teamB/JungJin/app.js

```diff
@@ -48,6 +48,30 @@ app.post('/users', function(req, res){
     res.status(201).json({message: "userCreated"});
 })
 
+app.get('/posts', function(req, res){
+    const query = myDataSource.query(`SELECT 
+    id,
+    title,
+    content,
+    imageurl,
+    user_id,
+    created_at,
+    updated_at 
+    FROM posts`, (err, rows) => {
+        res.status(200).json({posts : rows});
+    })
+})
+
+app.post('/posts', function(req, res){
+    const {title, content, user_id} = req.body
+    const query = myDataSource.query(`INSERT INTO posts(
+    title,
+    content,
+    user_id    
+    ) VALUES (?, ?, ?)`, [title, content, user_id])
```

Comment:
INSERT INTO 이후에 추가로 작업하는게 없기 때문에, query 변수에 담을 필요가 없어 보입니다~

```suggestion
    const { title, content, user_id } = req.body

    database.query(`
        INSERT INTO posts(
            title,
            content,
            user_id    
        ) VALUES (?, ?, ?)`, 
        [title, content, user_id])
```